### PR TITLE
remove Netty classes from OkHttpInvocationBuilder

### PR DESCRIPTION
**Pull request by bsideup**

### CODE REVIEW

File: core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java [hunk 1]
```diff
@@ -286,49 +285,98 @@ protected Response execute(OkHttpClient okHttpClient, Request request) {
...
+        @Override
+        public void accept(BufferedSource source) {
+            try {
+                while (!source.exhausted()) {
+                    // See https://docs.docker.com/engine/api/v1.37/#operation/ContainerAttach
+                    if(!source.request(HEADER_SIZE)) {
+                        return;
+                    }
+                    StreamType streamType = streamType(source.readByte());
+                    source.skip(3);
```
[the remark this answers is not in the document]
Comment (kiview) [reply core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java]: Ah, I was looking at the wrong example, found it now.
`header := [8]byte{STREAM_TYPE, 0, 0, 0, SIZE1, SIZE2, SIZE3, SIZE4}`

[the remark this answers is not in the document]
Comment (kiview) [reply core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java]: Can we maybe add this example as a code comment? Makes it much clearer if already available in the local context of the code.

[the remark this answers is not in the document]
Comment (bsideup) [reply core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java]: I think the link + code is already descriptive enough :)
I mean..... it is as simple as "read byte, parse as type, skip 3, read int" :) The example is actually less understandable to me, wtf is `SIZE{1-4}` :D

[the remark this answers is not in the document]
Comment (rnorth) [reply core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java]: Yeah, the link in the code was enough for me and made sense 😆

[the remark this answers is not in the document]
Comment (kiview) [reply core/src/main/java/org/testcontainers/dockerclient/transport/okhttp/OkHttpInvocationBuilder.java]: Alright, if you both agree :stuck_out_tongue:

**Comment by rnorth** [comment]
Released for preview in [1.9.0-rc2](https://github.com/testcontainers/testcontainers-java/releases/tag/1.9.0-rc2), to be published on Bintray.

**Pull request merged**